src/github.com/getlantern/flashlight/client/client.go

```diff
@@ -33,8 +33,22 @@ var (
 	// UIAddr is the address at which UI is to be found
 	UIAddr string
 
-	addr      = eventual.NewValue()
-	socksAddr = eventual.NewValue()
+	addr                = eventual.NewValue()
+	socksAddr           = eventual.NewValue()
+	proxiedCONNECTPorts = []int{
```

Comment:
I just moved this here since it caused some clumsy code and because it doesn't seem likely we'll change this via config versus auto-updates